Framework/Algorithms/inc/MantidAlgorithms/ReflectometryWorkflowBase.h

```diff
@@ -52,18 +52,18 @@ class DLLExport ReflectometryWorkflowBase : public API::DataProcessorAlgorithm {
                    Mantid::API::MatrixWorkspace_sptr>;
   using OptionalInteger = boost::optional<int>;
 
-  /// Convert the input workspace to wavelength, splitting according to the
-  /// properties provided.
+  /// Splits the input workspace into detector and monitor workpsaces according
+  /// to the properties provided.
   DetectorMonitorWorkspacePair
   toLam(Mantid::API::MatrixWorkspace_sptr toConvert,
         const std::string &processingCommands,
         const OptionalInteger monitorIndex, const MinMax &wavelengthMinMax,
         const OptionalMinMax &backgroundMinMax, const double &wavelengthStep);
```

Comment:
Could we rename this method? It no longer deals with wavelength conversion (that is now done outside) and can cause confusion in the future for new developers working in this area. The same happens with: 

`ReflectometryWorkflowBase::toLamMonitor()` 
and 
`ReflectometryWorkflowBase::toLamDetector()`

`toLam()` could be renamed to something like `splitDetectorsMonitors()` (as you have indicated in the description above). I am not sure about the names of the other two, perhaps something like:

`ReflectometryWorkflowBase::createDetectorWS()`
and
`ReflectometryWorkflowBase::createMonitorWS()`

would work.